spec/request/user_birth_record_spec.rb

```diff
@@ -129,7 +127,7 @@
         it 'is a no-op' do
```

Comment:
The description is inaccurate here now

Reply:
re-instated the old functionality that throws no errors if you're revoking a share that doesn't exist (or you have no access to).
Thiis means we need to also skip authorization (you can't ask pundit to authorize on a record that doesn't exist)

I'm not sure why we support this kind of request tho.